jep/0000/README.adoc

```diff
@@ -0,0 +1,315 @@
+6. SIG's are automatically notified of newly labelled issues by an link:https://github.com/jenkins-infra/helpdesk/blob/main/.github/workflows/autonotifier.yaml[autonotifier].
+7. SIG's are encouraged to run regular triage sessions to keep on top of the issues.
+
+=== LTS backporting
+
+See link:https://www.jenkins.io/download/lts/#backporting-process[Backporting Process] for the Jira process.
+
+If a pull request requires backporting a pull request will be created cherry-picking the change into the `stable-x` line.
+This will be automatically labelled as `into-lts`.
+
+It will be reviewed against the standard LTS criteria and either accepted and merged or rejected and closed.
+
+Pull requests that need to be backported but are not ready yet, should have the label `lts-candidate` applied to them.
+There will be reviewed during the LTS backporting window and the release lead will backport any remaining labelled pull requests.
+
+After a backport pull request is created the `lts-candidate` label will be removed, unless it was rejected but still eligible in a future backport.
+
+=== Documentation
+
+Contributing guides in Jenkins core and jenkins.io document that Jenkins core components use GitHub issues.
+
+== Motivation
+
+Jira is too disconnected from the code of Jenkins and the other core components.
+
+Historically most large open source projects used tools like bugzilla and Jira. Now there are many that use GitHub:
+- https://github.com/apple/swift
+- https://github.com/kubernetes
+- https://github.com/golang/go
+- https://github.com/rust-lang/rust
+
+The swift organization migrated in https://forums.swift.org/t/swift-bugs-are-moving-to-github-issues-and-we-need-your-help/56125
+
+The key reasons to migrate away are:
+
+=== Developer and user experience
+
+GitHub conveys a lot more information about the status of issues, pull requests and releases than Jira does.
+When an issue or pull request link is pasted you can see whether it is open/closed or merged without having to open it.
+On clicking a commit you can see all the releases it appears on.
+
+When a PR is merged in GitHub it can automatically close an issue.
+
+A developer is already in GitHub and doesn't have to load another system. Jira is manually maintained in respect of:
+- status
+- links to work
+- released in
+The data quality in Jira is lower than in GitHub due to it not getting as updated as often as GitHub.
+
+=== Jira server end of life February 2024
+
+Atlassian announced the link:https://www.atlassian.com/migration/assess/journey-to-cloud[end of life of Jira server] in February of 2021.
+Open source projects may continue being sponsored by creating a link:https://www.atlassian.com/software/views/open-source-license-request[Jira cloud request].
+
+Jira cloud has link:https://support.atlassian.com/jira-cloud-administration/docs/explore-jira-cloud-plans/[limitations] that would affect us around:
```

Comment:
This is a long page and it is unclear which limitations you are referring to.

Reply:
there's a list below this with the specific issues, (not detailed but I can expand if it would help)